core/src/main/java/org/infinispan/container/offheap/OffHeapDataContainer.java

```diff
@@ -587,7 +587,7 @@ public void executeTask(KeyValueFilter<? super WrappedBytes, ? super WrappedByte
 
    @Override
    public Iterator<InternalCacheEntry<WrappedBytes, WrappedBytes>> iterator() {
-      long time = timeService.time();
+      long time = timeService.wallClockTime();
```

Comment:
@wburns Have you ever tried running the entire core test suite with trace logging enabled? That does it for me almost every time.